docs/recipes/gulp/public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
need license. Can copy the html one from e.g. https://github.com/GoogleChrome/lighthouse/blob/e822cbea89634449d66d50243fba2adce094f1d0/lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html